src/server/llmQueries.ts

```diff
@@ -49,145 +61,183 @@ type LanguageCode = "nn" | "nb" | "en";
+          ${DEBUG_INSTRUCTION.nn}
+          ${ERROR_INSTRUCTION.nn}
         </Instructions>`,
 };
 
 type ReflectionQuery = Record<LanguageCode, (text: string) => string>;
 
 const REFLECTION_QUESTION_QUERY: ReflectionQuery = {
-  nb: (text: string) => `<draft>${text}</draft>
+  nb: (text: string) => `
+        <draft>${text}</draft>
         <Instructions>
-          Du har lang erfaring fra utdanningssektoren. Du er spesialist i å analysere fagartikler for å finne gode refleksjonsspørsmål som får elevene til å tenke gjennom hva de har lest.
-          Du har fått som oppdrag å lese gjennom artikkelen i <draft> og skrive fem refleksjonsspørsmål basert på artikkelen på NB. Spørsmålene skal være på et nivå som passer for elever på videregående skole.
-          Svaret skal leveres med refleksjoner på hvorfor du har valgt å inkludere de punktene du har valgt først. Deretter skal refleksjonsspørsmålene gis som punktliste i en <answer>-tag.
-          Om du ikke klarer å levere svaret på formatet <answer> som er spesifisert skal første linje i responsen være <ERROR>-tag.
+          Du har lang erfaring fra utdanningssektoren. 
+          Du er spesialist i å analysere fagartikler for å finne gode refleksjonsspørsmål som får elevene til å tenke gjennom hva de har lest.
+          Du har fått som oppdrag å lese gjennom artikkelen i <draft> tagget og skrive fem refleksjonsspørsmål basert på artikkelen på NB. 
+          Spørsmålene skal være på et nivå som passer for elever på videregående skole.
+          Spørsmålene gis som en punktliste inne i en <answer>-tag
+          ${DEBUG_INSTRUCTION.nb}
+          ${ERROR_INSTRUCTION.nb}
         </Instructions>`,
-  en: (text: string) => `<draft>${text}</draft>
+  en: (text: string) => `
+        <draft>${text}</draft>
         <Instructions>
-          You have extensive experience from the educational sector. You are specialized in analyzing educational articles to find good reflection questions which enables the students to reflect on what they have read.
-          Your task is to read through the article in <draft> and write five reflection questions based on the article in NB. The questions must be on a level which fits students in high school.
-          The response must be given with reflections as to why you have chosen to include the selected bullet points first. Then, the reflection questions must be given as bulletpoints in an <answer>-tag.
-          If you cannot deliver the answer in the given format <answer>, the first line in the response should be <ERROR>-tag.
+          You have extensive experience from the educational sector. 
+          You are specialized in analyzing educational articles to find good reflection questions which enables the students to reflect on what they have read.
+          Your task is to read through the article in <draft> and write five reflection questions based on the article in NB. 
```

Comment:
Jeg skjønner ikke helt greia her. Skal artikkelen være på bokmål, eller skal svaret være på bokmål? Sikkert greiest å fjerne all ambiguitet her så roboten ikke blir forvirret.

Reply:
Svaret skal være på språket til det queryen er. Så dette ble sikkert lagt inn for å sikre at innholdet i <draft> taggene ikke påvirker svaret.

Reply:
Skjønner ikke svaret ditt. Jeg lurte egentlig på hvorfor du hadde skrevet "NB" her. Skal det ikke være "NB"?